azure-pipelines.yml

```diff
@@ -162,9 +172,14 @@ jobs:
       testResultsFiles: $(Agent.HomeDirectory)/test-results.xml
       testRunTitle: Windows $(atom_channel)
     condition: succeededOrFailed()
-  - powershell: npm run report:coverage
+  - powershell: npm run report:coverage -- --build="Windows $(atom_channel)"
```

Comment:
I notice this looks inconsistent with what is shown above, where the `--build` flag is used with `npm run codecov` and not `npm run report:coverage`. Is this because Windows is special? Or is this unintentional?

Reply:
Totally unintentional copy/paste fail. Thanks for spotting this and for the fix 😅